### PR TITLE
chore: redirect handling

### PR DESCRIPTION
**Pull request by mfw78**
# Description

This PR handles redirections for legacy URLs.

# Changes

- [x] URLs that have moved permanently are sent `301`
- [x] URLs that have moved temporarily (as subsequent content is forthcoming) are sent `307`

All URLs use an absolute, domain excluded path per [vercel's redirect documentation](https://vercel.com/docs/projects/project-configuration#redirects-examples).

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]
I haven't checked each URL 🙈

**Review by alfetopito — COMMENTED** [review]
I checked it contains the same number of links as the spreadsheet, not the actual contents.

And I checked one redirect. It works as advertised.

**Review by alfetopito — APPROVED** [review]

File: vercel.json [hunk 1]
```diff
@@ -1,3 +1,595 @@
...
+        },
+        {
+            "source": "/cow-sdk/order-meta-data-appdata",
+            "destination": "https://learn.cow.fi/tutorial/simple-app-data",
+            "statusCode": 301
+        },
+        {
+            "source": "/cow-sdk/querying-the-cow-subgraph",
+            "destination": "https://learn.cow.fi/tutorial/getting-started-order",
+            "statusCode": 301
```
Comment (anxolin) [inline vercel.json]: This should be 302 as discussed here https://cowservices.slack.com/archives/C04N05LJK54/p1704884465781959?thread_ts=1704827135.936669&cid=C04N05LJK54

**Review by anxolin — COMMENTED** [review]
Nice! I will review one by one in the doc. For now just this feedback

File: vercel.json (hunk 1 above)
Comment (mfw78) [reply vercel.json]: The status codes / redirects were chosen per `vercel` documentation / recommendations.

**Pull request merged**